docs/single-demo.js

```
@@ -1,57 +1,64 @@
 "use strict";
 
 import React from "react";
-import ReactDOM from "react-dom";
+import { createRoot } from "react-dom/client";
 import Slider from "../src/slider";
```

ESLint

[error] 'Slider' is defined but never used.

(no-unused-vars)